packages/ui/src/components/common/ErrorWell.tsx

```diff
@@ -17,6 +19,11 @@ const ErrorWell: React.FC<Props> = ({ error, hasTxHashes }) => {
       !error?.message
     ) {
       return 'Oops! Something went wrong while processing your transaction.'
+    } else if (
+      error?.name &&
+      error?.message?.includes('does not support chain')
```

Comment:
Since fromChain is an optional prop, do we want to check that it is provided before we show this message?